build/patches/Timezone-Override.patch

```diff
@@ -0,0 +1,1379 @@
+         // TODO(crbug.com/735110): Figure out if this is the correct thing to do - here we are
+         // effectively treating non-ALLOW values as BLOCK.
+-        int index = (value == ContentSettingValues.SESSION_ONLY ? 2 : (value == ContentSettingValues.ALLOW ? 0 : 1));
+         listPreference.setValueIndex(index);
+     }
+ 
+diff --git a/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/SiteSettings.java b/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/SiteSettings.java
+--- a/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/SiteSettings.java
++++ b/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/SiteSettings.java
+@@ -120,7 +120,7 @@ public class SiteSettings
+             } else if (Type.SOUND == prefCategory && !checked) {
+                 p.setSummary(ContentSettingsResources.getSoundBlockedListSummary());
+             } else if (requiresTriStateSetting) {
+-                p.setSummary(ContentSettingsResources.getCategorySummary(setting));
++                p.setSummary(ContentSettingsResources.getCategorySummary(contentType, setting));
+             } else {
+                 p.setSummary(ContentSettingsResources.getCategorySummary(contentType, checked));
+             }
+diff --git a/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/SiteSettingsCategory.java b/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/SiteSettingsCategory.java
+--- a/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/SiteSettingsCategory.java
++++ b/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/SiteSettingsCategory.java
+@@ -42,7 +42,7 @@ public class SiteSettingsCategory {
+             Type.BACKGROUND_SYNC, Type.BLUETOOTH, Type.BLUETOOTH_SCANNING, Type.CAMERA,
+             Type.CLIPBOARD, Type.COOKIES, Type.DEVICE_LOCATION, Type.JAVASCRIPT, Type.MICROPHONE,
+             Type.NFC, Type.NOTIFICATIONS, Type.POPUPS, Type.PROTECTED_MEDIA, Type.SENSORS,
+-            Type.SOUND, Type.USB, Type.VIRTUAL_REALITY, Type.USE_STORAGE})
++            Type.SOUND, Type.USB, Type.VIRTUAL_REALITY, Type.USE_STORAGE, Type.TIMEZONE_OVERRIDE})
+     @Retention(RetentionPolicy.SOURCE)
+     public @interface Type {
+         // Values used to address array index - should be enumerated from 0 and can't have gaps.
+@@ -70,10 +70,11 @@ public class SiteSettingsCategory {
+         int BLUETOOTH = 19;
+         int VIRTUAL_REALITY = 20;
+         int USE_STORAGE = 21; // Always last as it should appear in the UI at the bottom.
++        int TIMEZONE_OVERRIDE = 22;
```

Comment:
Let's invert this with `21`, so that the comment still holds true?

Reply:
it is an untrue comment, in other parts of the code they refer to the order of the list that has never been decided by anyone, and is left to chance.
As I told you, I think that code is pretty dirty, I think many have got their hands on it.

Reply:
Sure, but by leaving it as-is we are not improving the situation. I would respect the comment since we do not plan to fix it.